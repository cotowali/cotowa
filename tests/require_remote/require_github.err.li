require 'github:cotowali/cotowali/main/tests/__missing__'
