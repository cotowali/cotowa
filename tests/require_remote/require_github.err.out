error: require_github.err.li 1,1: faild to get https://raw.githubusercontent.com/cotowali/cotowali/main/tests/__missing__ (404 Not Found)
    1 | require 'github:cotowali/cotowali/main/tests/__missing__'
      | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
